packages/blade/src/components/Table/docs/BasicTable.stories.tsx

```diff
@@ -119,6 +119,8 @@ const TableTemplate: StoryFn<typeof TableComponent> = ({ ...args }) => {
       <TableComponent
         {...args}
         data={data}
+        defaultSelectedIds={['1', '3']}
+        onSelectionChange={console.log}
```

Comment:
I don't see this reflected in story 🤔 

https://61c19ee8d3d282003ac1d81c-pghwdoruck.chromatic.com/?path=/story/components-table--table&args=selectionType:multiple

Reply:
oh got it got it. It works when you select "multiple" then refresh the page

Reply:
Yeah since its `default`, it'll only take the values on first render